tests/test_datasets/test_dataset_functions.py

```diff
@@ -407,6 +410,85 @@ def test__getarff_path_dataset_arff(self):
         self.assertIsInstance(arff_path, str)
         self.assertTrue(os.path.exists(arff_path))
 
+    def test__download_minio_file_object_does_not_exist(self):
+        self.assertRaisesRegex(
+            FileNotFoundError,
+            r"Object at .* does not exist",
+            _download_minio_file,
+            source="http://openml1.win.tue.nl/dataset20/i_do_not_exist.pq",
```

Comment:
Is this a new test server?

Reply:
There is currently no distinction between a test and production `minio` server. Will need updating once that's sorted out 😔